app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,28 @@
+class InvitesController < ApplicationController
+  def new; end
+
+  def create
+    if !handle_in_system?
```

Comment:
I'm confused how that handle is able to indicate whether they have an email address. Would you mind explaining this? (Can also do it in a call if that is easier)